pbs/pbsrequest.go

```diff
@@ -71,6 +84,7 @@ type AdUnit struct {
 	Bids       []Bids           `json:"bids"`
 	ConfigID   string           `json:"config_id"`
 	MediaTypes []string         `json:"media_types"`
+	Video      PBSVideo         `json:"video"`
```

Comment:
Since PBSVideo is our "internal" type, I somehow thought we should not expose it in AdUnit. Please correct me if I am wrong.

Reply:
I'm not sure there is really a distinction here. We need a way to pass `video` attributes on the request and this already has everything defined. I guess I don's see the need for the protocol between client <=> Prebid Server to be different than Prebid Server <=> ORTB endpoint in this case. Let me know if you had other thoughts though.